jinja/index.html

```diff
@@ -153,7 +153,7 @@ <h5>Follow the gradient</h5>
                 <img class="img-fluid" src="img/globe.svg" width="70px"></img><br>
                 <h5>Makes PyTorch and TensorFlow quantum</h5>
             </div>
-            <p class="grey-text">Support for hybrid quantum and classical models, and compatible with existing machine learning libraries. Quantum circuits can be set up to interface with either NumPy, PyTorch, or TensorFlow, allowing hybrid CPU-GPU-QPU computations.
+            <p class="grey-text">Support for hybrid quantum and classical models, and compatible with existing machine learning libraries. Quantum circuits can be set up to interface with either NumPy, PyTorch, JAX or TensorFlow, allowing hybrid CPU-GPU-QPU computations.
```

Comment:
```suggestion
            <p class="grey-text">Support for hybrid quantum and classical models, and compatible with existing machine learning libraries. Quantum circuits can be set up to interface with either NumPy, PyTorch, JAX, or TensorFlow, allowing hybrid CPU-GPU-QPU computations.
```

This feels like an oxford comma type of sentence